transports/sqs/operations-scripting_create_sqs_[5,).partial.md

```diff
@@ -132,4 +138,97 @@ sqs-transport endpoint subscribe [name] [event-type]
 
 `-r` | `--region`: Overrides the environment variable 'AWS_SECRET_ACCESS_KEY'
 
+`-p` | `--prefix`: Prefix to prepend to the topic provisioned for the event type and the subscribing queue
+
+### sqs-transport endpoint set-policy events
+
+Set the IAM policy on the input queue of an endpoint, based on the event types the endpoint subscribed to, using:
+
+```
+sqs-transport endpoint set-policy [name] events 
+                                         [--event-type]
+                                         [--access-key-id]
+                                         [--secret]
+                                         [--region]
+                                         [--prefix]
+```
+
+#### arguments
+
+`name` : Name of the endpoint (required)
+
+#### options
+
+`-evt` | `--event-type` : Full name of the event allow through the IAM policy (e.g. MyNamespace.MyMessage), can be repeated multiple times to allow multiple event types to pass.
+ 
+`-i` | `--access-key-id` : Overrides the environment variable 'AWS_ACCESS_KEY_ID'
+
+`-s` | `--secret` : Overrides the environment variable 'AWS_REGION'
+
+`-r` | `--region`: Overrides the environment variable 'AWS_SECRET_ACCESS_KEY'
+
+`-p` | `--prefix`: Prefix to prepend to the topic provisioned for the event type and the subscribing queue
+
+### sqs-transport endpoint set-policy wildcard
+
+Set the IAM policy on the input queue of an endpoint, based on wildcard conditions, using:
+
+```
+sqs-transport endpoint set-policy [name] wildcard
+                                         [--account-condition]
+                                         [--namespace-condition]
+                                         [--prefix-condition]
+                                         [--remove-event-type]
+                                         [--access-key-id]
+                                         [--secret]
+                                         [--region]
+                                         [--prefix]
+```
+
+#### arguments
+
+`name` : Name of the endpoint (required)
+
+#### options
+
+`-ac` | `--account-condition` : Allow all messages from any topic in the account to pass. No value needs to be provided, the account name will be derived from the endpoint input queue.
+
+`-pc` | `--prefix-condition` : Allow all messages from any topic with prefix to pass. No value needs to be provided, prefix from the `-p | --prefix` option will be used.
+
+`-nc` | `--namespace-condition` : Allow all messages from any message in the specified namespaces to pass.
+
+`-revt` | `--remove-event-type` : Existing event type conditions on the policy will not be removed by default. Specify this option to remove an existing event type conditions, can be repeated multiple times to remove multiple event types.
```

Comment:
```suggestion
`-revt` | `--remove-event-type` : Existing event type conditions on the policy will not be removed by default. Specify this option to remove an existing event type conditions in case they are covered by the wildcard policy implicitely. Can be repeated multiple times to remove multiple event types.
```